### ISSUE TITLE
claude 3

### ISSUE THREAD

**Issue by morandalex** [continuedev/continue#926]
### Before submitting your bug report

- [X] I believe this is a bug. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [ ] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue) that reports the same bug
- [ ] I've seen the [troubleshooting guide](https://continue.dev/docs/troubleshooting) on the Continue Docs

### Relevant environment info

```Markdown
it seems that cannot use  claude 3 models

with this config I get 


   {
      "model": "claude-3-opus-20240229",
      "contextLength": 2048,
      "title": "Claude-3",
      "apiKey": "<apikey>",
      "completionOptions": {},
      "provider": "anthropic"
    }



```
Error streaming response: Error: max_tokens (4096) is too close to context_length (2048), which doesn't leave room for chat. This would cause incoherent responses. Try increasing the context_length parameter of the model in your config file.
```

instead with this config 


```json
   {
      "model": "claude-3-opus-20240229",
      "contextLength": 8000,
      "title": "Claude-3",
      "apiKey": "<apikey>",
      "completionOptions": {},
      "provider": "anthropic"
    }
```

I get 
```
Error streaming response: Error: {"type":"error","error":{"type":"invalid_request_error","message":"\"claude-3-opus-20240229\" is not supported on this API. Please use the Messages API instead."}
```
I am using ubuntu 20
same error is generated from jetbrain ide and visual studio code
```


### Description

_No response_

### To reproduce

_No response_

### Log output

_No response_

**Comment by sestinj** [comment]
@morandalex Fixing this right now, I'll let you know this morning as soon as we have a release!

**Comment by sestinj** [comment]
@morandalex this is now released in 0.9.79
